docs/content/foundations/layout-positioning/layout-grid.mdx

```diff
@@ -0,0 +1,356 @@
+	<div className="tablet:grid-col-8">.tablet:grid-col-8</div>
+	<div className="grid-col-6 tablet:grid-col-4">.col-6 .tablet:grid-col-4</div>
+</div>
+<!-- Columns start at 50% wide on mobile and bump up to 33.3% wide on desktop -->
+<div className="grid-row">
+	<div className="grid-col-6 tablet:grid-col-4">.col-6 .tablet:grid-col-4</div>
+	<div className="grid-col-6 tablet:grid-col-4">.col-6 .tablet:grid-col-4</div>
+	<div className="grid-col-6 tablet:grid-col-4">.col-6 .tablet:grid-col-4</div>
+</div>
+<!-- Columns are always 50% wide, on mobile and desktop -->
+<div className="grid-row">
+	<div className="grid-col-6">.col-6</div>
+	<div className="grid-col-6">.col-6</div>
+</div>
+```
+
+## Offsetting Columns
+
+`.grid-offset-[1-12]` offsets an item by the specified number of grid columns as shown in the following example.
+
+<GridColExample />
+
+```html
+<div className="grid-row">
+	<div className="grid-col-8 grid-offset-4">.grid-col-8.grid-offset-4</div>
+</div>
+```
+
+## Column Wrapping
+
+Items wrap when the sum of the grid columns is more than 12 as shown in the following example.
+
+```html
+<div className="grid-row">
+	<div className="grid-col-8">.grid-col-8</div>
+	<div className="grid-col-3">.grid-col-3</div>
+	<div className="grid-col-5">.grid-col-5</div>
+</div>
+```
+
+<h2 id="gutters">Gutters</h2>
+
+### Default Gutter
+
+Add `grid-gap` to a grid row to add a gap (or gutter) between each column in the row. The default gap width is 2 units and 4 units at `desktop` and higher. Customize the width of the gap by adjusting the value of `$theme-column-gap` in project settings as shown in the following example.
+
+<GridColExample />
+
+```html
+<div className="grid-row grid-gap">
+	<div className="grid-col-4"><div>.grid-col-4</div></div>
+	<div className="grid-col-4"><div>.grid-col-4</div></div>
+	<div className="grid-col-4"><div>.grid-col-4</div></div>
+</div>
+```
+
+### Large Gutter
+
+`.grid-gap-lg` adds a larger gap (or gutter) between each column in a row than `grid-gap`. The default large-gap width is 32px (4 spacing units). Customize the width of the large gap by adjusting the value of `$theme-column-gap-lg` in project settings. There is also a `.grid-gap-sm` (2px) set with `$theme-column-gap-sm`. Also, you can add the following system values with `grid-gap`:
+
+- `.grid-gap-2px`
+- `.grid-gap-05`
+- `.grid-gap-1`
+- `.grid-gap-2`
+- `.grid-gap-3`
+- `.grid-gap-4`
+- `.grid-gap-5`
+- `.grid-gap-6`
+
+<GridColExample />
+
+```html
+<div className="grid-row grid-gap-lg">
+	<div className="grid-col-4"><div>.grid-col-4</div></div>
+	<div className="grid-col-4"><div>.grid-col-4</div></div>
+	<div className="grid-col-4"><div>.grid-col-4</div></div>
+</div>
+```
+
+## Sass Mixins
+
+When generating your CSS from USWDS source files, you have the option of customizing many system defaults by modifying project theme variables. USWDS also provides grid mixins for adding grid functionality to custom semantic component CSS.
```

Comment:
We should look for all instances of USWDS and change to NCIDS. (With a note that the NCIDS grid uses the USWDS grid at the top of this file.)